src/vs/workbench/contrib/search/browser/media/searchview.css

```diff
@@ -42,6 +42,7 @@
 	max-height: 134px;
 }
 
+/* Warning the height is also use in searchWidget.ts as a constant*/
```

Comment:
two minor suggestions:
* is also use**d**
* replace 'Warning the height' with 'NOTE: height'

Reply:
yes, good idea and thanks for the spelling error